SU-SWS/ace-sdssgryphon
# SDSS-471: moving the extra utility text to in a template

# READY FOR REVIEW

# Summary
- Moved the area called a utility menu out of Javascript and into a template.
- Hid the text from the version of the site that should not have it.
- This is a step to fix the bug and move towards getting all the customization out of the Javascript. 

# Review By (Date)
- 1/26/23

# Urgency
- High. It is a bug and it would be good to get it out of javascript.

# Steps to Test

1. Clone this site `blt drupal:sync --site=esys` and `blt drupal:sync:files --site=esys`
2. Verify that on the /people that clicking the load more does not replicate the top utility area.
3. Change the lockup to `none` and verify the utility area does not appear on that version of the header. 

# Affected Projects or Products
- SDSS subtheme

# Associated Issues and/or People
[- SDSS-471](https://stanfordits.atlassian.net/browse/SDSS-471)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
